enarx-keep-sgx-shim/src/main.rs

```diff
@@ -21,6 +21,59 @@ mod handler;
 use handler::Handler;
 use nolibc::x86_64::syscall::Number as SysCall;
 use sgx_types::{ssa::StateSaveArea, tcs::Tcs};
+use crt0stack::{Builder, Handle, OutOfSpace, Entry};
+
+// Hard-code the entry point for now.
+//
+// I'd like to improve this situation in the future. However, we cannot let
```

Comment:
Let's file a debt issue for this and link to it here.

Reply:
Done.